tasks/configure.yml

```diff
@@ -5,15 +5,20 @@
 - set_fact: archivematica_src_configure_ss_api_key={{ 999999999999999999999 | random | to_uuid | hash('md5') }}
   when: "archivematica_src_configure_ss_api_key is undefined and archivematica_src_configure_ss|bool"
 
-# Check whether SS user exists
-- name:  Check SS default test user
+- name: "Check for default SS user"
   become: "yes"
-  command: sqlite3  "{{archivematica_src_ss_environment['SS_DB_NAME']}}" "select username from auth_user where (id=1 and last_login is NULL and first_name='' and last_name='' and email='x@x.com' and username='test');"
+  shell: echo 'select username from auth_user where (id=1 and last_login is NULL and first_name="" and last_name="" and email="x@x.com" and username="test");'  | {{ archivematica_src_ss_virtualenv }}/bin/python manage.py dbshell
+  args:
+    chdir: "{{ archivematica_src_ss_app }}"
```

Comment:
Hi @scollazo we have an old typo on this variable:
```
maml@molinillo ~/external-roles/artefactual-archivematica-qa-src $ grep -Ri archviematica_src_ss_app *
tasks/configure.yml:    chdir: "{{ archviematica_src_ss_app }}"
tasks/configure.yml:    app_path: "{{ archviematica_src_ss_app }}"
tasks/ss-main.yml:      dest: "{{ archviematica_src_ss_app }}"
tasks/ss-main.yml:    app_path: "{{ archviematica_src_ss_app }}"
tasks/ss-db.yml:    app_path: "{{ archviematica_src_ss_app }}"
tasks/ss-db.yml:    app_path: "{{ archviematica_src_ss_app }}"
templates/etc/systemd/system/archivematica-storage-service.service.j2:WorkingDirectory={{ archviematica_src_ss_app }}
vars/main.yml:archviematica_src_ss_app: "/usr/lib/archivematica/storage-service"
``` 
So we can continue using the ` archviematica_src_ss_app` var or fix the typo.